common/azure/src/main/kotlin/com/cosmotech/api/azure/CsmAzureBasedSecurityConfiguration.kt

```diff
@@ -18,16 +18,23 @@ import org.springframework.security.oauth2.jwt.Jwt
 import org.springframework.stereotype.Component
 
 @Component
-@ConditionalOnProperty(name = ["csm.platform.vendor"], havingValue = "azure", matchIfMissing = true)
+@ConditionalOnProperty(
+    name = ["csm.platform.identityProvider.code"], havingValue = "aad", matchIfMissing = true)
 internal class CsmAzureBasedSecurityConfiguration(
     private val csmPlatformProperties: CsmPlatformProperties,
 ) {
 
   @Bean
-  fun jwtAuthenticationConverter(): Converter<Jwt, out AbstractAuthenticationToken> =
-      AADJwtBearerTokenAuthenticationConverter(
-          csmPlatformProperties.authorization.principalJwtClaim,
-          AADResourceServerProperties.DEFAULT_CLAIM_TO_AUTHORITY_PREFIX_MAP)
+  fun aadJwtAuthenticationConverter(): Converter<Jwt, out AbstractAuthenticationToken> {
+    val claimAuthorityMap =
+        AADResourceServerProperties.DEFAULT_CLAIM_TO_AUTHORITY_PREFIX_MAP.toMutableMap()
+
+    csmPlatformProperties.azure?.activeDirectory?.claimToAuthorityPrefix?.let {
+      claimAuthorityMap.putAll(it)
+    }
+    return AADJwtBearerTokenAuthenticationConverter(
```

Comment:
allowed-tenants for okta from iss claim too?

  @Bean
  fun jwtDecoder(): JwtDecoder {
disappeared?